.gitignore

```diff
@@ -19,3 +19,5 @@ src/test/data/sandbox/
 
 # MacOS custom attributes files created by Finder
 .DS_Store
+
+/bin/
```

Comment:
Is there any particular reason why you need to ignore this folder? Are you using this folder in your command?

Reply:
for some reason vscode creates 200 files in the bin folder when opening the project in that editor. i can remove this change